x-pack/packages/ml/aiops_components/src/document_count_chart/document_count_chart.tsx

```diff
@@ -94,14 +86,24 @@ export interface DocumentCountChartProps {
     fieldFormats: FieldFormatsStart;
     uiSettings: IUiSettingsClient;
   };
-  /** Optional callback function which gets called the brush selection has changed */
-  brushSelectionUpdateHandler?: (windowParameters: WindowParameters, force: boolean) => void;
+  /**
+   * Optional callback function which gets called the brush selection has changed
+   *
+   * @param windowParameters Baseline and deviation time ranges.
+   * @param force Force update
+   * @param logRateAnalysisType `spike` or `dip` based on median log rate bucket size
+   */
+  brushSelectionUpdateHandler?: (
```

Comment:
Suggestion to export a `BrushSelectionUpdateHandler` type so that it can be used in other interfaces that has brushSelectionUpdateHandler as a an arg.

Reply:
Updated in f6380024d41.